test/powershell/Modules/Microsoft.PowerShell.Management/Get-ComputerInfo.Tests.ps1

```diff
@@ -1332,11 +1332,6 @@ try {
             }
         }
 
-        It "Verify that alias 'gin' exists" {
```

Comment:
The idea (from @lzybkr) is that we should put all alias tests in one place (Default-Aliases.Tests.ps1) rather than spread on many files.